documentation/modules/creating-plan-wizard-290-cnv.adoc

```diff
@@ -0,0 +1,130 @@
+
+* *Use new storage map*: Allows you to create one or two new storage maps by supplying the following data. These maps are attached to this plan, which is then their owner. For each map, specify the following.
+
+** *Source storage* : Select from the list.
+** *Target storage* : Select from the list.
++
+If needed, click *Add mapping* to add an additional mapping.
+** *Storage map name*: Enter a name or let {project-short} automatically generate a name for the network map.
+
+. Click *Next*.
++
+// Other settings page
+. In the *Other settings (optional)* page, you have the option to change the *Transfer network* of your migration plan. 
++
+The transfer network is the network used to transfer the VMs to {virt}. This is the default transfer network of the provider. 
+
+** Verify that the transfer network is in the selected target namespace.
+
+** To choose a different transfer network, select a different transfer network from the list.
+** Optional: To configure an additional {ocp-short} network in the {ocp-short} web console, click *Networking > NetworkAttachmentDefinitions*.
++
+To learn more about the different types of networks {ocp-short} supports, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/{ocp-version}/html-single/networking/index#additional-networks-provided_understanding-multiple-networks[Additional Networks in OpenShift Container Platform].
+
+** To adjust the maximum transmission unit (MTU) of the {ocp-short} transfer network, you must also change the MTU of the VMware migration network. For more information, see xref:selecting-migration-network-for-vmware-source-provider_vmware[Selecting a migration network for a VMware source provider].
+
+. Click *Next*.
++
+// Hooks page
+. In the *Hooks (optional)* page, you can add a pre-migration hook, a post-migration hook, or both types of migration hooks. All are optional.
+
+. To add a hook, select the appropriate *Enable hook* checkbox.
+. Enter the *Hook runner image*.
+. Enter the *Ansible playbook* of the hook in the window.
++
+[NOTE]
+====
+You cannot include more than one pre-migration hook or more than one post-migration hook in a migration plan.
+====
+
+. Click *Next*.
++
+// Review and create page
+. In the *Review and Create* page, review the information displayed.
+. Edit any item by doing the following:
+
+.. Click its *Edit step* link.
++ The wizard opens to the page on which you defined the item.
+.. Edit the item.
+.. Either click *Next* to advance to the next page of the wizard or click *Skip to review* to return directly to the *Review and create* page.
+
+. When you finish reviewing the details of the plan, click *Create plan*. {project-short} validates your plan. 
+. After you correct all problems, click *Skip to review* and then click *Create plan*.
```

Comment:
Feel likes the beginning of Step 22 is redundant with step 20 c. I would maybe delete that first sentence "After you correct all problems, click Skip to review and then click Create plan."